PrimerSDK/Classes/User Interface/Primer/MultiCardIconComponent.swift

```diff
@@ -7,6 +7,7 @@
 
 import UIKit
 
+// FIXME: Unused?
```

Comment:
Most likely unused, did you comment it out and it was fine? (if so let's remove)

Reply:
Removed